can-bus/json/schema_tx.json

```diff
@@ -0,0 +1,71 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"patternProperties": {
+		"(FSM|BMS|DCM|DIM|GSM|PDM)_[A-Z]\\w+": {
+			"type": "object",
+			"properties": {
+				"msg_id": {
+					"type": "integer",
+					"description": "CAN Message Group ID Value"
+				},
+				"cycle_time": {
+					"type": "integer",
+					"description": "Amount of times per second this message should update"
+				},
+				"signals": {
+					"type": "object",
+					"description": "All of the individual messages in this group",
+					"patternProperties": {
+						"[A-Z]\\w+": {
+							"type": "object",
+							"properties": {
+								"resolution": {
+									"description": "Finest resolution that this message requires, determines amount of bits that are required",
+									"type": "number"
+								},
+								"bits": {
+									"description": "Explicit Declaration of amount of bits required for CAN Message",
+									"type": "integer"
+								},
+								"min": {
+									"description": "Minimum Value this CAN Message can take on",
+									"type": "integer"
+								},
+								"max": {
+									"description": "Maximum Value this CAN Message can take on",
+									"type": "integer"
+								},
+								"unit": {
+									"description": "Units that are used to measure this message, (Optional)",
+									"type": "string"
+								},
+								"scale": {
+									"description": "idk scale?",
+									"type": "integer"
+								},
+								"offset": {
+									"description": "idk offset?",
+									"type": "integer"
+								},
```

Comment:
Comment should be: 
1. Scale: "Explicitly specify the DBC scale parameter used to pack/unpack this signal.
2. Offset: "Explicitly specify the DBC offset parameter used to pack/unpack this signal.

Reply:
Also, all of these that are currently "integer" should be type "number" (except bits should stay integer)

Reply:
Also, these parameters all specify values for the signals, not messages. Can you update all the descriptions to reflect this?